src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -99,7 +106,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
     const conversation = this.props.conversation
     const artwork = conversation.artworks[0]
 
-    const partnerName = conversation.to_name
+    const galleryName = conversation.to.name
```

Comment:
Yeah it should always be `partnerName` because partners aren't always galleries.